website/docs/r/google_project_organization_policy.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: "google"
+page_title: "Google: google_project_organization_policy"
+sidebar_current: "docs-google-project-organization-policy"
+description: |-
+ Allows management of Organization policies for a Google Project.
+---
+
+# google\_project\_organization\_policy
+
+Allows management of Organization policies for a Google Project. For more information see
+[the official
+documentation](https://cloud.google.com/resource-manager/docs/organization-policy/overview) and
+[API](https://cloud.google.com/resource-manager/reference/rest/v1/projects/setOrgPolicy).
+
+## Example Usage
+
+To set policy with a [boolean constraint](https://cloud.google.com/resource-manager/docs/organization-policy/quickstart-boolean-constraints):
+
+```hcl
+resource "google_project_organization_policy" "serial_port_policy" {
+  project    = "123456789"
```

Comment:
I think the resource wants the project id right, not the number? If so, mind doing something with text in it so people don't think they have to figure out what their project number is?